triage/2023-12-26.md

```diff
@@ -0,0 +1,207 @@
+| Improvements ✅ <br /> (secondary) | -0.4% | [-0.6%, -0.3%] | 5     |
+| All ❌✅ (primary)                 | -0.7% | [-0.7%, -0.7%] | 2     |
+
+* This PR only modified the UI and tests of rustdoc, so this is just noise.
+
+Make closures carry their own ClosureKind [#119258](https://github.com/rust-lang/rust/pull/119258) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=2271c26e4a8e062bb00d709d0ccb5846e0c341b9&end=1ab783112ab4e4807304dbd249b39771246013ef&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | -     | -              | 0     |
+| Regressions ❌ <br /> (secondary)  | -     | -              | 0     |
+| Improvements ✅ <br /> (primary)   | -1.1% | [-1.1%, -1.1%] | 1     |
+| Improvements ✅ <br /> (secondary) | -     | -              | 0     |
+| All ❌✅ (primary)                 | -1.1% | [-1.1%, -1.1%] | 1     |
+
+* This is probably just noise.
+
+#### Mixed
+
+add more niches to rawvec [#106790](https://github.com/rust-lang/rust/pull/106790) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=f704f3b93b1543cf504ecca0052f9f8531b1f61f&end=51c0db6a9118735190466d15432b5d8890bb09ee&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 0.4%  | [0.1%, 0.7%]   | 20    |
+| Regressions ❌ <br /> (secondary)  | 0.6%  | [0.2%, 1.0%]   | 2     |
+| Improvements ✅ <br /> (primary)   | -0.4% | [-2.0%, -0.1%] | 27    |
+| Improvements ✅ <br /> (secondary) | -0.9% | [-5.3%, -0.2%] | 16    |
+| All ❌✅ (primary)                 | -0.1% | [-2.0%, 0.7%]  | 47    |
+
+* This PR enabled more optimization opportunities for LLVM, which could in theory help with
+program optimization, but it also caused some binary size increases and compile-time regressions.
+* Not marking as triaged yet.
+
+Remove metadata decoding DefPathHash cache [#119265](https://github.com/rust-lang/rust/pull/119265) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=08cc634f1a26f30801daa1cbe7b866b12aaf1edd&end=ebb821f625db3cbc77c675326165b8dfc3d7a2be&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 0.9%  | [0.9%, 0.9%]   | 1     |
+| Regressions ❌ <br /> (secondary)  | -     | -              | 0     |
+| Improvements ✅ <br /> (primary)   | -0.6% | [-2.1%, -0.2%] | 101   |
+| Improvements ✅ <br /> (secondary) | -1.2% | [-3.2%, -0.2%] | 65    |
+| All ❌✅ (primary)                 | -0.6% | [-2.1%, 0.9%]  | 102   |
+
+* Very nice check and incremental wins across the board.
+* The single regression is probably just noise returning back to steady state.
```

Comment:
It looks like it has sustained for a bit, interestingly enough, rather than being a simple noise swing. But it's fine, it's strangely been fixed by now by https://github.com/rust-lang/rust/pull/119258, and the situation is better than the previous steady state.... 🤷

Reply:
Oh, it seems that I interpreted these two exact PRs as start of noise - end of noise :laughing: In any case, it's not worth to investigate on this PR, which otherwise had so many wins.

Reply:
Agreed that there's no need to look into it at all, it's almost noise like and is already fixed anyways.